config/puma.rb

```diff
@@ -1,56 +1,35 @@
+# This configuration file will be evaluated by Puma. The top-level methods that
```

Comment:
Most of these changes are from overwriting puma.rb (originally generated back when we were on Rails 5) with the puma.rb from a freshly generated Rails 7 app.